src/pages/Dogs/AdoptionLogNavbar.jsx

```diff
@@ -55,50 +57,54 @@
               type="text"
               placeholder="Search"
               size="sm"
-              value={search}
-              onChange={e => setSearch(e.target.value)}
+              value={searchDog}
+              onChange={e => setSearchDog(e.target.value)}
               className={styles.customSelectInput}
             />
           </InputGroup>
         </div>
-          <select
-            value={filter}
-            border="1px solid black"
-            onChange={e => setFilter(e.target.value)}
-            className={styles.customSelectInput}
-          >
-            <option value="">Filter by...</option>
-            <option value="all">All</option>
-            <option value="service">Service</option>
-            <option value="therapy">Therapy</option>
-            <option value="special-needs">Special Needs</option>
-            <option value="staff-adoption">Staff Adoption</option>
-            <option value="deceased">Deceased</option>
-            <option value="all-males">All Males</option>
-            <option value="all-females">All Females</option>
-          </select>
-          <div className={styles['mobile-container']}>
-            <label className={styles.label}>
-              Facility:
-              <select value={facility} onChange={e => setFacility(e.target.value)} className={styles.customSelectInput}>
-                <option value="">All</option>
-                <option value="facility-1">OC Probation</option>
-                <option value="facility-2">Theo Lacy Facility</option>
-                <option value="facility-3">OC Sheriffs Department</option>
-              </select>
-            </label>
-            <Button size="md" onClick={() => setSelectAll(true)} style={{ marginRight: '10px' }}>
-              Select All
-            </Button>
-            <Button
-              size="md"
-              aria-label="Export"
-              rightIcon={<DownloadIcon />}
-              onClick={() => setExportData()}
+        <select
+          value={filter}
+          border="1px solid black"
+          onChange={e => setFilter(e.target.value)}
+          className={styles.customSelectInput}
+        >
+          <option value="">Filter by...</option>
+          <option value="all">All</option>
+          <option value="service">Service</option>
+          <option value="therapy">Therapy</option>
+          <option value="special-needs">Special Needs</option>
+          <option value="staff-adoption">Staff Adoption</option>
+          <option value="deceased">Deceased</option>
+          <option value="all-males">All Males</option>
+          <option value="all-females">All Females</option>
+        </select>
+        <div className={styles['mobile-container']}>
+          <label className={styles.label}>
```

Comment:
🚫 **[eslint]** <[jsx-a11y/label-has-associated-control](https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/label-has-associated-control.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>A form label must be associated with a control.